modules/plan/README.md

```diff
@@ -0,0 +1,96 @@
+<!-- BEGIN MODULE HOOK -->
```

Comment:
Can you remover the `<!-- BEGIN MODULE HOOK -->` - it does nothing

Reply:
Resolved in latest commit